apps/web/src/components/forms/montitor-form.tsx

```diff
@@ -505,11 +514,16 @@ export function MonitorForm({
           )}
         />
         <div className="sm:col-span-full">
-          {/*
-           * We could think of having a 'double confirmation' one,
-           * to check if the endpoint works and approve afterwards
-           * and confirm anyways even if endpoint failed
-           */}
+          {pingFailed && (
```

Comment:
You don't need the ternary operation here as we use `open={pingFailed}` to open the Confirmation Modal.